scripts/_common.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#=================================================
+# COMMON VARIABLES
+#=================================================
+
+# dependencies used by the app
+pkg_dependencies="deb1 deb2"
```

Comment:
?

Reply:
removed ^^, kept it for the day it will be needed ^^